docker/build/cd/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# escape=`
+
+ARG PARENT_IMAGE
+ARG SOLUTION_IMAGE
+
+FROM ${SOLUTION_IMAGE} as solution
+FROM ${PARENT_IMAGE}
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+WORKDIR c:\module\cd\content
```

Comment:
**question:** This would be the custom directory on the develepors machine, correct? Maybe worth moving to docker-compose?

Reply:
This directory contains the necessary files for UrlMapper and it needs to be stored in the image. In the customer project we will copy this folder from the UrlMapper image to the cd/cm image.